tests/element_info_test.py

```diff
@@ -48,15 +48,20 @@ def test_section_definitions_from_multiple_sources(dpf_server):
         files, server=dpf_server, default_unit_system=unit_systems.solver_nmm
     )
 
+    ud_mat_id = composite_model.material_names["Epoxy Carbon UD (230 GPa) Wet"]
```

Comment:
We cannot rely on dpf mat ids in tests. Dpf mat ids were different when using a windows grpc server.